examples/app-playwright/tests/basic.test.ts

```diff
@@ -10,7 +10,7 @@ test.use({
 })
 */
 
-test('test', async ({ page }) => {
-  await page.goto('/')
+test('test', async ({ page, goto }) => {
+  await goto('/', { waitUntil: 'hydration' })
```

Comment:
would this behaviour also work with `page.goto`?

Reply:
Unfortunately not, we don't recommend overriding page.goto so I think exposing it as a fixture is a nice middle-line between both sides (overriding class vs. making this extra functionality available).